src/pages/AuthPage.js

```diff
@@ -1,81 +1,83 @@
-import AuthForm, {STATE_LOGIN} from 'components/AuthForm';
+import AuthForm, { STATE_LOGIN } from 'components/AuthForm';
 import React from 'react';
-import {connect} from 'react-redux';
-import {Card, Col, Row} from 'reactstrap';
-import {authenticateUser} from '../actions/authenticationActions';
-import {Redirect} from "react-router-dom";
-import {isExpired, setWebCookie} from './../utils/jwtTokenUtils'
+import { connect } from 'react-redux';
+import { Card, Col, Row } from 'reactstrap';
+import { authenticateUser } from '../actions/authenticationActions';
+import { Redirect } from 'react-router-dom';
+import { isExpired, setWebCookie } from './../utils/jwtTokenUtils';
 
 class AuthPage extends React.Component {
+  handleAuthState = authState => {
+    if (authState === STATE_LOGIN) {
+      this.props.history.push('/login');
+    } else {
+      this.props.history.push('/signup');
+    }
+  };
 
-    handleAuthState = authState => {
-        if (authState === STATE_LOGIN) {
-            this.props.history.push('/login');
-        } else {
-            this.props.history.push('/signup');
-        }
-    };
+  handleLogoClick = () => {
+    this.props.history.push('/');
+  };
 
-    handleLogoClick = () => {
-        this.props.history.push('/');
-    };
+  handleLoginSubmit = authentication => {
+    this.props.dispatch(
+      authenticateUser(authentication.username, authentication.password),
+    );
+  };
 
-    handleLoginSubmit = (authentication) => {
-        this.props.dispatch(authenticateUser("admin@communis.com", "A@12word"));
```

Comment:
@jobinthomasnypl, the main idea of .env was to remove this.

Reply:
@maxjerin Can you please resolve the conflicts?